.github/workflows/base_setup/action.yaml

```diff
@@ -0,0 +1,51 @@
+name: 'openpilot env setup'
+
```

Comment:
let's keep this one called "setup", it's not really a base setup

Reply:
Done